miso-web/src/main/webapp/pages/adminsub.jsp

```diff
@@ -30,11 +30,6 @@
 				Export</a></li>
 	</ul>
 
-	<h2>Sequencing</h2>
-	<ul class="bullets">
-		<li><a href="<c:url value="/miso/pools/ready"/>">Ready to Run</a></li>
```

Comment:
I think the lab will still want quick access to which pools are in line to be sequenced next, but that might be better done by replacing "Ready to Run" with "Outstanding Orders". I've added an agenda item to discuss this.